src/interface/src/app/plan/plan-routing.module.ts

```diff
@@ -11,13 +11,7 @@ const routes: Routes = [
     title: 'Plan Details',
     component: PlanComponent,
     canActivate: [AuthGuard],
-
     children: [
-      {
```

Comment:
I say we leave this route, even though they can be a bit redundant, it makes it clear that config and config/id are technically two different paths on the app

Reply:
Ok, let me revert that change ;)

Reply:
DONE